src/App.js

```diff
@@ -173,11 +174,12 @@ function App() {
   };
 
   return (
-    <div className='App'>
-      <ThemeProvider theme={appliedTheme}>
-        <CssBaseline />
-        <AppBar position='static' className={classes.appBar}>
-          <Toolbar>
+    <ThemeProvider theme={appliedTheme}>
+      <CssBaseline />
+
+      <AppBar position="static" className={classes.appBar}>
+        <Container maxWidth='lg'>
```

Comment:
Great usage Container component. You opened my eyes to a new opportunities.
Thanks @mohamednajiub